### ISSUE TITLE
Avoid specific orientations

### ISSUE THREAD

**Issue by joaocabogon** [atenpas/gpd#46]
Hey @atenpas. 
Which parameters in the tutorial1.launch file could one change to avoid grasps where the binormal is in the vertical direction for instance?

**Comment by atenpas** [comment]
I don't think this is possible directly. Yet, you could write a simple filter (check dot product of binormal and vertical direction) and add it to `grasp_detector.cpp` such that it is used **after** grasp candidates have been generated.
